### DISCUSSION TITLE
Enforced coding standards

### DISCUSSION THREAD

**Discussion by SindreLindahl** [frangoteam/FUXA#1330] [General]
Noticed on readme it mentions coding standards (indents, braces). Why is this asked for? Isn't this an automatic job for linters and formatters? Put that stuff in CI-pipeline and then each person can add vscode settings (format on save) and never have to think about it?

Just looked weird to mention this on main readme.

**Comment by unocelli** [comment]
Hi sorry for the delay. You are right, we had a bit of a fight with lint, feel free to do a PR with lint's configuration
